internal/aws/containerinsight/utils_test.go

```diff
@@ -866,3 +867,14 @@ func TestConvertToOTLPMetricsForPodContainerStatusMetrics(t *testing.T) {
 	md = ConvertToOTLPMetrics(fields, tags, zap.NewNop())
 	checkMetricsAreExpected(t, md, fields, tags, expectedUnits)
 }
+
+func TestHostProcessContainer(t *testing.T) {
+	os.Setenv(RunInContainer, "True")
```

Comment:
we should unify around the constant or unify just around the string "True".  I think the string "True" is good enough